apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.js

```diff
@@ -47,6 +76,9 @@
 
     quill.setContents(quill.clipboard.convert({ html: contents }));
 
+    const getText = () => quill.getText();
+    const counter = options.counter === 'none' ? null : new Counter(options.counter, uuid, getText);
```

Comment:
Nitpicking:
```suggestion
    const counter = options.counter === 'none' ? null : new Counter(options.counter, uuid, quill.getText);
```

Reply:
The closure is necessary to bind `quill` - Inlining it breaks the call to getText (since this is unbound).

Reply:
Ah, good point, I missed that.